CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ We're so excited you're interested in helping with SuperTokens! We are happy to
 ## Development Setup  
```

Comment:
an extension to this PR - can you add a PR check for github to run the pre commit hook like it's done in other repos?

Reply:
Done